src/main/resources/swagger/api-docs.yaml

```diff
@@ -424,7 +424,7 @@ paths:
       responses:
         '200':
           description: Proxy connection successful
-        '401':
+        '403':
```

Comment:
It's odd that the 401/403 is only documented on these `/notebooks/...` APIs. Maybe we should include them on the `/api/cluster/...` APIs since those can throw 403 as well.